modules/ivc_champva/spec/models/vha_10_7959c_shared_examples.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.shared_examples 'form model 10_7959C' do |form_number|
+  let(:data) do
```

Comment:
I noticed all of the model specs use a custom `let(:data)` instead of using the [JSON fixture data](https://github.com/department-of-veterans-affairs/vets-api/tree/master/modules/ivc_champva/spec/fixtures/form_json). I kept things consistent here and did the same thing; but should we use the fixtures instead of these custom lets?

I don't think this PR should address this issue but I can create a new ticket to refactor all of the model specs to use the JSON fixtures instead